_i18n/es/resources/user-guides/vps_run_node.md

```diff
@@ -2,48 +2,46 @@
 {% include disclaimer.html translated="false" version=page.version %}
```

Comment:
finaly

Reply:
Got it haha I'm sorry, I just copy-pasted it thinking it was a script for other thing, what should I change in version? Gonna read Readme again but I think this is new.